### PR TITLE
Automation: Disabled addOns job updateAddOns option

### PR DESCRIPTION
**Pull request by psiinon**
Signed-off-by: Simon Bennetts <psiinon@gmail.com>

### CODE REVIEW

File: addOns/automation/src/main/java/org/zaproxy/addon/automation/gui/AddOnJobDialog.java [hunk 1]
```diff
@@ -58,7 +58,8 @@ public AddOnJobDialog(AddOnJob job) {
 
         this.addTextField(NAME_PARAM, this.job.getData().getName());
 
-        this.addCheckBoxField(UPDATE_ADDONS_PARAM, this.job.getParameters().getUpdateAddOns());
+        // Disabled until this can work without breaking things
```
Comment (thc202) [inline addOns/automation/src/main/java/org/zaproxy/addon/automation/gui/AddOnJobDialog.java:61]: Could you add a task tag? (e.g. `XXX` or `FIXME`)

File: addOns/automation/src/test/java/org/zaproxy/addon/automation/jobs/AddOnJobUnitTest.java [hunk 2]
```diff
@@ -190,6 +191,7 @@ void shouldReturnFileConfigData() {
     }
 
     @Test
+    @Disabled // Disabled which the updateAddons option is disabled.
```
Comment (thc202) [inline addOns/automation/src/test/java/org/zaproxy/addon/automation/jobs/AddOnJobUnitTest.java:194]: `@Disabled("The updateAddons option is disabled.")`

Comment (psiinon) [reply addOns/automation/src/test/java/org/zaproxy/addon/automation/jobs/AddOnJobUnitTest.java]: Ah ha - didnt know you could do that...

File: addOns/automation/src/main/java/org/zaproxy/addon/automation/gui/AddOnJobDialog.java (hunk 1 above)
Comment (psiinon) [reply addOns/automation/src/main/java/org/zaproxy/addon/automation/gui/AddOnJobDialog.java]: Done :)

**Review by thc202 — APPROVED** [review]

**Comment by thc202** [comment]
Thank you!

**Review by kingthorin — APPROVED** [review]

**Pull request merged**